redbot/core/core_commands.py

```diff
@@ -1579,20 +1579,23 @@ async def set_showsettings(self, ctx: commands.Context):
         global_data = await ctx.bot._config.all()
         locale = global_data["locale"]
         regional_format = global_data["regional_format"] or _("Same as bot's locale")
+        colour = discord.Colour(global_data["color"])
 
         prefix_string = " ".join(prefixes)
         settings = _(
             "{bot_name} Settings:\n\n"
             "Prefixes: {prefixes}\n"
             "{guild_settings}"
             "Locale: {locale}\n"
-            "Regional format: {regional_format}"
+            "Regional format: {regional_format}\n"
+            "Default colour: {colour}"
```

Comment:
How about calling this default embed colour? This setting is specifically for embeds as far as I know (and the help of `[p]set colour` seems to agree with me on that)
```suggestion
            "Default embed colour: {colour}"
```